install.sh

```diff
@@ -0,0 +1,276 @@
+fi
+
+
+#===================================#
+# Navigator Dependency Installation #
+#===================================#
+
+instlog "Installing Navigator dependencies from the Ubuntu repositories"
+sudo apt-get install -qq libompl-dev
+sudo apt-get install -qq python-pygame
+
+instlog "Installing Navigator ROS dependencies"
+sudo apt-get install -qq ros-indigo-roboteq-driver
+sudo apt-get install -qq ros-indigo-rosserial
+sudo apt-get install -qq ros-indigo-rosserial-python ros-indigo-rosserial-arduino
+
+instlog "Installing Navigator dependencies from source"
+rm -rf /tmp/pyode-build
+mkdir -p /tmp/pyode-build
+cd /tmp/pyode-build
+sudo apt-get build-dep -qq python-pyode
+sudo apt-get remove -qq python-pyode
+apt-get source --compile -qq python-pyode
+sudo dpkg -i python-pyode_*.deb
+
+
+#==============================#
+# Sub8 Dependency Installation #
+#==============================#
+
+instlog "Installing Sub8 dependencies from the Ubuntu repositories"
+sudo apt-get update -qq
+sudo apt-get install -qq cmake binutils-dev python-pip
+sudo apt-get install -qq python-scipy python-pygame python-numpy python-serial
+sudo apt-get install -qq python-twisted socat
+sudo apt-get install -qq libvtk5-dev python-vtk
+sudo apt-get install -qq tesseract-ocr
+sudo apt-get install -qq libboost-all-dev python-dev python-qt4-dev python-qt4-gl python-opengl freeglut3-dev libassimp-dev
+sudo apt-get install -qq libpcl-1.7-all libpcl-1.7-all-dev
+sudo apt-get install -qq libompl-dev
+sudo apt-get install -qq libusb-1.0-0-dev
+
+instlog "Installing Sub8 ROS dependencies"
+sudo apt-get install -qq ros-indigo-gazebo7-msgs ros-indigo-gazebo7-ros ros-indigo-gazebo7-plugins ros-indigo-gazebo7-ros-control
+sudo apt-get install -qq ros-indigo-control-toolbox ros-indigo-controller-manager ros-indigo-transmission-interface ros-indigo-joint-limits-interface ros-indigo-hardware-interface
+sudo apt-get install -qq ros-indigo-sophus
+sudo apt-get install -qq ros-indigo-driver-base
+sudo apt-get install -qq ros-indigo-camera-info-manager
+sudo apt-get install -qq ros-indigo-spacenav-node
+sudo apt-get install -qq ros-indigo-camera1394
+sudo apt-get install -qq ros-indigo-stereo-image-proc
+sudo apt-get install -qq ros-indigo-pcl-ros ros-indigo-pcl-conversions
+sudo apt-get install -qq ros-indigo-rosbag-image-compressor ros-indigo-compressed-image-transport ros-indigo-compressed-depth-image-transport
+
+instlog "Installing Sub8 dependencies from Python PIP"
+sudo pip install -q -U setuptools
+sudo pip install -q -U mayavi > /dev/null 2>&1
+sudo pip install -q -U argcomplete
+sudo pip install -q -U scikit-learn > /dev/null 2>&1
+sudo pip install -q -U tqdm
+sudo pip install -q -U service_identity
+sudo pip install -q -U pyasn1
+sudo pip install -q -U characteristic
+sudo pip install -q -U crc16
+
+instlog "Getting Sub8 ROS packages we need to install from source"
+ros_git_get https://github.com/uf-mil/Sub8.git
```

Comment:
mayavi was used for visualizing on some sub thresholding tool, I think so that one is not necessary. The other ones I'd say are useful to have except maybe crc16.